src/nav-base.yml

```diff
@@ -295,3 +295,31 @@
     children:
     - title: Sunsetting npm Enterprise
       url: /sunsetting-npm-enterprise
+
+- title: Policies
+  url: /policies
+  children:
+  - title: Terms of Use
+    url: /policies/terms
```

Comment:
Is the duplication necessary here (with the child having `url: /policies/blah` when the parent already has `url: /policies`)?